WebRoot/js/zimbraMail/prefs/view/ZmFilterRuleDialog.js

```diff
@@ -1550,7 +1550,8 @@ function(inputs, conf, field) {
 		if (!date) {
 			return null;
 		}
-		return String(date.getTime() / 1000);
+		AjxDateUtil._serverDateFormatter = new AjxDateFormat("yyyyMMdd");
+		return AjxDateUtil._serverDateFormatter.format(new Date(date));
```

Comment:
Can we use `AjxDateUtil.getServerDate` ?
https://github.com/Zimbra/zm-ajax/blob/develop/WebRoot/js/ajax/util/AjxDateUtil.js#L598